app/lib/search_query_builder.rb

```diff
@@ -65,7 +65,23 @@ def metadata_fields
   end
 
   def order_query
-    keywords ? order_by_relevance_query : default_order_query
+    sort_options = finder_content_item.dig('details', 'sort')
+
+    if sort_options.present?
+      sort_option = if params['order']
+                      sort_options.detect { |option| option['name'].parameterize == params['order'] }
+                    end
+
+      sort_option ||= sort_options.detect { |option| option['default'] } || { 'key' => default_order }
+
+      if %w(relevance -relevance).include?(sort_option['key'])
+        order_by_relevance_query
+      else
+        { 'order' => sort_option['key'] }
+      end
+    else
+      keywords ? order_by_relevance_query : default_order_query
+    end
```

Comment:
Fixed with 5f35b94

Reply:
This is confusing because we're supporting ordering by either the `sort` or `default_order` (deprecated) attributes of the finder.

If the sort option is present we want to sort by it, unless the key is `relevance` or `-relevance`.